bundles/org.openhab.binding.kaleidescape/README.md

```diff
@@ -1,37 +1,32 @@
 # Kaleidescape Binding
 
-This binding is used to control and retrieve information from a Kaleidescape movie player.
-All movie player components including the original K-Player series, M Class Players, Cinema One, Alto, and Strato are supported.
-The 4 zone audio only KMUSIC-4000 is not supported at this time.
-As there are many good control options already available for these components, this binding focuses primarily on retrieving information
- for display purposes and to use in rules for controlling other Things such lighting, projector lens control, masking, etc.
-Basic playback transport controls are provided and any other command that is supported by the control protocol can be sent to the component through rules based commands.
-See [Kaleidescape-System-Control-Protocol-Reference-Manual.pdf](https://support.kaleidescape.com/article/Control-Protocol-Reference-Manual) for a reference of available commands.
-To simplify the design of the binding code, a different Thing instance is created for each component
- in a multi-zone system and each Thing maintains its own socket connection to the target component.
-Overall this binding supports the majority of information and commands available in the Kaleidescape control protocol but is by no means exhaustive.
-Any feedback or suggestions for improvement are welcome.
+The Ultimate Movie Player - Kaleidescape elevates every component in your theater with higher fidelity
+source material for the ultimate cinematic experience.
 
-The binding supports two different kinds of connections:
-
-* direct IP connection (preferred),
-* serial connection (19200-8-N-1)
+This binding now makes it possible to easily integrate almost all of the capabilities of the Kaleidescape control protocol into openHAB.
+Beyond just integrating playback transport controls, all meta-data provided via the control protocol is made available for display purposes and to use in rules.
```

Comment:
are made

Reply:
I must have read this 20 times when I was writing it but still messed up the grammar. Fixed now and ready for your re-review.